doc/Makefile

```diff
@@ -1,20 +0,0 @@
-# Minimal makefile for Sphinx documentation
```

Comment:
Is there a reason to drop the Makefile? `make html` is how I'm used to building Sphinx docs locally - am I out of date?

Reply:
For the last question: Maybe? :P 

I tend to run `sphinx-build docs build/docs`, which I find straightforward-enough. I am down to add a tox target for the documentation build instead -- since realistically, there's both environment management *and* the documentation build command that are reasonably coupled. I'm ambivalent on the `make html` but if we do want to keep those, I'd prefer to file a follow up adding those back, as generated by the latest Sphinx version.